NationalSecurityAgency/emissary
# Dockerfile and docker compose updates

Dockerfile now has all docker flavors (ubi8 (default), centos7 and alpine)
Unnecessary Dockerfiles have been removed
docker compose files have been updated to support single emissary server and cluster option

To test:

Run through the docker steps in DEVELOPING.md starting at line 596

Make sure you can build a docker
Run emissary docker and process a file
Run docker compose for single emissary server and process file
Run docker compose for emissary cluster and process file
